src/sentry/static/sentry/app/views/incidents/details/body.tsx

```diff
@@ -127,7 +127,7 @@ const StyledPageContent = styled(PageContent)`
   }
 `;
 
-const StyledNavTabs = styled(NavTabs)`
+const StyledNavTabs = styled(NavTabs)<NavTabs['props']>`
```

Comment:
Hm, it should inherit these. Does it not?

Ah maybe because it's a function component.

Reply:
There were compiler errors before this was added.